brd-ios/breadwallet/src/ApplicationController.swift

```diff
@@ -431,8 +431,11 @@ class ApplicationController: Subscriber, Trackable {
         }
         
         homeScreen.didTapTrade = { [unowned self] in
-            let currencies = coreSystem.assetCollection?.allAssets.compactMap { $0.value.code } ?? []
-            coordinator?.showSwap(currencies: currencies)
+            // TODO: Remove when ready.
+//            let currencies = coreSystem.assetCollection?.allAssets.compactMap { $0.value.code } ?? []
```

Comment:
this can be removed, instead use the currency pair worker to get swap pairs

Reply:
Will do that in the future prs.